src/app/Login/login.jsx

```diff
@@ -0,0 +1,73 @@
+import React, { Component } from "react";
+import Avatar from "react-avatar";
+import firebase from "firebase";
+import database from "../../index.js";
+import { connect } from "react-redux";
+import "./login.css";
+import { authentication } from "../../store/modules/auth/actions";
+
+class Login extends Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {};
+  }
+
+  componentDidMount() {
+    firebase.auth().onAuthStateChanged(user => {
+      if (user) {
+        this.setState({ user });
+        this.props.authentication(user);
+        database.ref("users").push({
```

Comment:
@jyotigautam This is will store user to firebase database every time any user logins to the system.
Please refer the 'storeUser' in this file on github https://github.com/aviabird/angularhunt/blob/master/src/app/services/authentication.service.ts 
And Implement the system accordingly.